app/views/qsets/_modal_new_qset.html.erb

```diff
@@ -14,6 +14,12 @@
             <input type="text" class="form-control" name="name" />
           </div>
           <div class="form-group">
+            <label for="text">Qset Type: </label>
+            Questions<input type="radio" name="qset_type" value="questions" checked>
```

Comment:
I noticed this is a bit squished at 100% in my Mac Chrome. Any way we can space these out a bit (maybe using CSS? some bootstrap styling for radio buttons on forms?)

Also, typically browsers want the label to match a specific input. That way, if you click on a label, it selects the input (in this case, the radio button). Otherwise, you have to click on the radio button itself, which is less user-friendly.

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#Using_the_for_attribute and http://www.w3schools.com/tags/tag_label.asp.